docs/photoniq/prerendering/best-practices-prerendering.md

```diff
@@ -5,26 +5,22 @@ title: Prerendering Best Practices
 
 Prerendering is an optimization technique that generates static HTML pages from dynamic content to improve website performance. The following best practices help you get the most out of Prerendering.
 
-## Selective Prerendering
-
-To maximize the benefits of prerendering, target specific areas of your website that will gain the most from this optimization technique. Typically, these are high-traffic pages or sections featuring dynamic content that seldom changes. By focusing on these areas, you ensure efficient use of resources and faster load times for your users.
-
 ## Domain Configuration
 
 Be very careful when configuring the domains for prerendering. Incorrect settings can result in caching problems and the delivery of outdated content. Make sure the specified domains are accurate and correspond to the sections of your site that you intend to prerender.
```

Comment:
```suggestion
Be very careful when configuring the origins for prerendering. Incorrect settings can result in prerendering problems and the delivery of very outdated content. Ensure the specified origins settings are accurate and correspond to your website's needs.
```

It is okay to serve outdated content. When you use the caching feature, you know that the pages won't be up to date. The problem comes when, for example, your website is updated every week, and you configured the cache to expire in 30 days.

Reply:
It is important to mention the purge cache feature, so the user can remove the cache for particular pages in the case he wants to serve the most up to date version of the page.

Reply:
We mention the caching in the next best practice.

Reply:
Here my concern is with:
```
Make sure the specified origins are accurate and correspond to the sections of your site that you intend to prerender.
```
When you configure an origin, you can prerender whatever URL you want from that origin. The customer needs to make sure the configuration is OK and it is working fine for the website.
What about this?
```
Make sure the specified origin settings are accurate and correspond to your website's needs.
```